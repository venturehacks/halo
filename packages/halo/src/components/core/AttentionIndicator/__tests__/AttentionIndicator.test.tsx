import { render } from 'enzyme';
import React from 'react';

import { AttentionIndicator } from '../index';

describe('AttentionIndicator', () => {
  describe('snapshot', () => {
    describe('color', () => {
      test('orange', () => {
        const component = render(<AttentionIndicator color="orange" />);
        expect(component).toMatchSnapshot();
      });

      test('red', () => {
        const component = render(<AttentionIndicator color="red" />);
        expect(component).toMatchSnapshot();
      });
    });

    describe('shape', () => {
      test('circle', () => {
        const component = render(<AttentionIndicator shape="circle" />);
        expect(component).toMatchSnapshot();
      });

      test('oval', () => {
        const component = render(<AttentionIndicator shape="oval" />);
        expect(component).toMatchSnapshot();
      });
    });

    describe('size', () => {
      test('sm', () => {
        const component = render(<AttentionIndicator size="sm" />);
        expect(component).toMatchSnapshot();
      });

      test('md', () => {
        const component = render(<AttentionIndicator size="md" />);
        expect(component).toMatchSnapshot();
      });
    });
  });
});
