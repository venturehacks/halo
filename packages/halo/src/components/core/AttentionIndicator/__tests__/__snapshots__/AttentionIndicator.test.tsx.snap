// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AttentionIndicator snapshot color orange 1`] = `
<div
  class="component orange sm circle"
/>
`;

exports[`AttentionIndicator snapshot color red 1`] = `
<div
  class="component red sm circle"
/>
`;

exports[`AttentionIndicator snapshot shape circle 1`] = `
<div
  class="component red sm circle"
/>
`;

exports[`AttentionIndicator snapshot shape rounded-rectangle 1`] = `
<div
  class="component red sm roundedRectangle"
/>
`;

exports[`AttentionIndicator snapshot size md 1`] = `
<div
  class="component red md circle"
/>
`;

exports[`AttentionIndicator snapshot size sm 1`] = `
<div
  class="component red sm circle"
/>
`;
